packages/lambdas/jest.config.ts

```diff
@@ -0,0 +1,32 @@
+import * as path from "path";
+import type { JestConfigWithTsJest } from "ts-jest";
+
+const isE2E = process.env.E2E === "true";
+
+process.env.ENV_TYPE = "development";
+
+const cwd = process.cwd();
+const paths = [cwd, ...(cwd.includes("packages") ? [] : ["packages", "lambdas"])];
+const tsconfig = path.resolve(...paths, "tsconfig.cloud.json");
+
+const config: JestConfigWithTsJest = {
+  preset: "ts-jest",
+  testEnvironment: "node",
+  verbose: true,
+  testPathIgnorePatterns: ["/node_modules/", "/dist/"],
+  testMatch: isE2E
+    ? ["**/__tests__/**/(*.)+(spec|test).e2e.[jt]s?(x)"]
+    : ["**/__tests__/**/(*.)+(spec|test).[jt]s?(x)"],
+  setupFilesAfterEnv: ["./src/__tests__/env-setup.ts"],
```

Comment:
This is only needed if tests require env variables to be present to work, which I don't think its the case on this package (we don't even have the `.env.test` being included on `env-setup.ts`).